user_profile_modal: Add user ID

tanyasingh7 opened the pull request:
<!-- What's this PR for?  (Just a link to an issue is fine.) --> Fixes issue #21713. 

**Testing plan:** <!-- How have you tested? --> Passed all tests.

**GIFs or screenshots:** <!-- If a UI change.  See:
  https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
  -->
<img width="517" alt="Screen Shot 2022-04-07 at 1 20 17 PM" src="https://user-images.githubusercontent.com/60750284/162261632-a8b3d6c1-8636-4a84-9f1b-b7580aba37aa.png">

<img width="518" alt="Screen Shot 2022-04-13 at 7 21 24 PM" src="https://user-images.githubusercontent.com/60750284/163288699-c7e29339-a611-46d8-99b2-3c042e5d1fdf.png">




<!-- Also be sure to make clear, coherent commits:
  https://zulip.readthedocs.io/en/latest/contributing/version-control.html
  -->

alya commented:
Thanks for the PR!

Please:

1. Add a second screenshot that also shows the user email.
2. Modify the commit message and PR description following https://zulip.readthedocs.io/en/latest/contributing/version-control.html
3. Fix the failing test.
4. Post a comment here when the above have been completed. :)

tanyasingh7 commented:
Hi Alya, thank you for your feedback! I made all the changes you requested. I added a second screenshot above, modified the commit messages, and fixed the failing test. Please let me know if there is anything else I need to modify. Thank you!

alya commented:
The next step is to squash your commits into one. :)

tanyasingh7 commented:
Hi Alya, I've been trying to fix this issue using git rebase and it seems like on my end all the commits have been squashed into one but I haven't seen the pull request update to show these changes. Additionally, I have tried to push my updated branch but it won't allow me to until I fix merge conflicts (which I am unable to see) and I can't git pull because it says I have unmerged files. Do you think it would be best if I closed this PR and reopened a new one?

alya commented:
@tanyasingh7 feel free to do that, though please make sure to link to this PR so that the discussion above is not lost.
